hudi-common/src/main/java/org/apache/hudi/common/config/HoodieConfig.java

```diff
@@ -42,6 +42,7 @@ public class HoodieConfig implements Serializable {
 
   private static final Logger LOG = LoggerFactory.getLogger(HoodieConfig.class);
 
+  public static final String BASE_PATH_KEY = "hoodie.base.path";
```

Comment:
Forgot to mention, could we use `HoodieCommonConfig.BASE_PATH` instead?

Reply:
done